python/jupytercad_core/jupytercad_core/schema/__init__.py

```diff
@@ -11,7 +11,7 @@
 from .interfaces.chamfer import IChamfer  # noqa
 from .interfaces.fillet import IFillet  # noqa
 from .interfaces.any import IAny  # noqa
-
+from .interfaces.sketch import ISketchObject
```

Comment:
You might want to take a look at pre-commit failure for this or just add `# noqa` at the end of the line